test/common.py

```diff
@@ -2559,7 +2655,31 @@ def browser_open(cls, url):
 
     browser_args = shlex.split(browser_args)
     logger.info('Launching browser: %s', str(browser_args))
-    cls.browser_proc = subprocess.Popen(browser_args + [url])
+
+    with FileLock(browser_spawn_lock_filename) as count:
+      # Firefox is a multiprocess browser. Killing the spawned process will not
+      # bring down the whole browser, but only one browser tab. So take a delta
+      # snapshot before->after spawning the browser to find which subprocesses
+      # we launched.
```

Comment:
Does this comment only apply to windows?

Reply:
Yes, it seems to. Updated the comment.